libraries/chain/include/eosio/chain/trace.hpp

```diff
@@ -45,6 +69,7 @@ namespace eosio::chain {
       std::optional<fc::exception>    except;
       std::optional<uint64_t>         error_code;
       std::vector<char>               return_value;
+      std::optional<std::vector<sync_call_trace>>  call_traces;
```

Comment:
Why an optional vector? I'm guessing you don't want call_traces to appear in JSONified trace if an action didn't use sync calls at all?

Reply:
Yes, that is what I want. Should we show `call_traces` with an empty `""` instead if an action did not make any sync calls? @arhag

Reply:
Wouldn't an empty vector be sufficient to unambiguously indicate the action didn't use sync calls at all?

I don't see the harm of always including the `call_traces` in the JSON output of the action trace with it being `[]` if no sync calls were made.